### PR TITLE
NEXT-30352 - Add cluster setup config docs

### PR DESCRIPTION
**Pull request by keulinho**
See https://gitlab.shopware.com/shopware/6/product/platform/-/merge_requests/11389

### CODE REVIEW

File: guides/hosting/installation-updates/cluster-setup.md [hunk 1]
```diff
@@ -4,6 +4,20 @@ The setup of high-scaling systems differs from a normal installation of Shopware
 
 This guide contains information for everyone who intends to start with such a project.
 
+## Shopware configuration
+
+{% hint style="info" %}
+This configuration is available starting with Shopware version 6.5.6.0
+{% endhint %}
```
Comment (shyim) [inline guides/hosting/installation-updates/cluster-setup.md:11]: we should mention what it does.

Comment (keulinho) [reply guides/hosting/installation-updates/cluster-setup.md:11]: I added some explanations

Comment (JoshuaBehrens) [reply guides/hosting/installation-updates/cluster-setup.md:11]: I still don't get what it means. Like a cache clear of symfony cache files?

Comment (keulinho) [reply guides/hosting/installation-updates/cluster-setup.md:11]: Yes that is the first specific use case: https://github.com/shopware/platform/commit/1508ddc7d3d051de0a99f348d6abeeae00e534c1

but there may come more cases we might improve, that why i tried to keep it more generic ;) 

But i can add that as an example

**Review by JoshuaBehrens — APPROVED** [review]

**Review by shyim — APPROVED** [review]

**Comment by shyim** [comment]
@bojanrajh do we need to do there something to get this merged? as you added Manual migration label

**Comment by bojanrajh** [comment]
@shyim I'll merge this PR and run the migration scripts in `next` branch afterwards.

@sushmangupta We need to update our docs guidelines with the new vitepress syntax (`{% hint` becomes `:::`, ...).

**Pull request merged**